agent/stats/engine_integ_test.go

```diff
@@ -143,7 +146,7 @@ func TestStatsEngineWithNewContainersWithoutHealth(t *testing.T) {
 		},
 		testTask)
 
-	err = engine.MustInit(taskEngine, defaultCluster, defaultContainerInstance)
+	err = engine.MustInit(context.TODO(), taskEngine, defaultCluster, defaultContainerInstance)
```

Comment:
You should use a cancellable context here similar to other test files (applies throughout this file)

Reply:
ok, there are too many context creations in the test, I missed some, I will double check.